pyproject.toml

```
@@ -19,3 +19,6 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.scripts]
 pub_script = 'python_poetry_template.main:get_specific_bucket_objects'
+
+[tool.black]
+line-length = 120
```

Newline character - [POSIX 3.206 Line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)
```suggestion
line-length = 120
```
```suggestion
line-length = 120

```